lgtm.yml

```diff
@@ -0,0 +1,30 @@
+path_classifiers:
+  test: "regression-tests*"
+  library:
+  - "ext"
+  - exclude: ext/luawrapper
+  - exclude: ext/yahttp
+  - "docs"
+  - "pdns/recursordist/html/js"
+  - "pdns/dnsdistdist/src_js"
+  - "pdns/dnsdistdist/html/js"
+extraction:
+  cpp:
+    configure:
+      command:
+      - "pushd pdns/dnsdistdist"
+      - "sed -i '/^dist_man_MANS/d' Makefile.am"
+      - "autoreconf -vi && ./configure"
+      - "popd"
+      - "pushd pdns/recursordist/"
+      - "sed -i '/^dist_man_MANS/d' Makefile.am"
+      - "autoreconf -vi && ./configure"
+      - "popd"
+      - "sed -i 's/codedocs docs//' Makefile.am"
+      - "autoreconf -vi"
+      - "./configure --with-modules=\"bind\""
```

Comment:
I'm fairly sure that `--with-modules='bind gmysql gpgsql'` would just work: the dependencies will be detected and installed automagically by LGTM. I think you may have to use single quotes though.